lib/token.h

```diff
@@ -755,7 +755,7 @@ class CPPCHECKLIB Token {
     }
 
     bool isCMultiChar() const {
-        return (((mTokType ==  eChar) && isPrefixStringCharLiteral(mStr, '\'', emptyString)) &&
+        return (((mTokType ==  eChar) && isPrefixStringCharLiteral(mStr, '\'', emptyString)) && !isPrefixStringCharLiteral(mStr, '\'', "\'\\") &&
```

Comment:
this function should return true for `'\r\n'`, will it?

Reply:
Yes. I added test case for it